src/components/Regions/Region/Region.test.js

```diff
@@ -0,0 +1,28 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../../../Redux/createStore';
+import Region from './Region';
+
+describe('Region component test', () => {
+  test('Region snapshot', () => {
+    const props = {
+      name: 'region 1',
+      newCases: 50,
+    };
+    const {
+      name, newCases,
+    } = props;
+    const region = render(
+      <Router>
+        <Provider store={store}>
+          <Region
+            name={name}
+            newCases={newCases}
+          />
+        </Provider>
+      </Router>,
+    );
+    expect(region).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] When I run your tests (`npm run test`), they all fail. The reason they are failing is that you are importing the `Images` folder in your `apiFunction.js` file but it isn't mocked in your tests. This is the result I get when I run the test:

![image](https://user-images.githubusercontent.com/77106962/166109454-a2bc3467-d6d9-4751-98af-6917968f23ed.png)


#### Kindly ensure that all tests pass before you resubmit. 👍